cpp/test/CMakeLists.txt

```diff
@@ -255,6 +255,7 @@ if(BUILD_TESTS)
     NAME
     NEIGHBORS_TEST
     PATH
+    test/neighbors/ann_cagra/test_float_uint32_t.cu
```

Comment:
Do we need the other 3 files in `test/neighbors/ann_cagra` if we're only going to compile this one version?

Reply:
I have removed them (for now).